org/mozilla/jss/tests/JSS_SelfServServer.java

```diff
@@ -112,13 +113,14 @@ public void doIt(String[] args) throws Exception {
             if (args[3].equalsIgnoreCase("true") == true)
                 TestInetAddress = true;
             if (args.length >= 5)
-                port = new Integer(args[4]).intValue();
+                port = new Integer(Integer.valueOf(args[4]));
             if (args.length >=6 && args[5].equalsIgnoreCase("verbose")) {
                 bVerbose = true;
             }
             if (args.length >=7 && !args[6].equalsIgnoreCase("default")) {
                 fServerCertNick = args[6];
             }
+        } catch (NumberFormatException e) {
```

Comment:
Out of curiousity, why adding this line? Seems like it should be caught with the next line. :)